pkg/pillar/Dockerfile.in

```diff
@@ -39,8 +39,8 @@ RUN set -e && for patch in ../patches/*.patch; do \
 
 RUN [ -z "$GOARCH" ] || export CC=$(echo /*-cross/bin/*-gcc) ;\
     echo "Running go vet" && go vet ./... && \
-    echo "Running go fmt" && ERR=$(gofmt -e -l -s $(find . -name \*.go | grep -v /vendor/)) && \
-       if [ -n "$ERR" ] ; then echo "go fmt Failed - ERR: "$ERR ; exit 1 ; fi && \
+    echo "Running go fmt" && ERR=$(gofmt -e -d -s $(find . -name \*.go | grep -v /vendor/)) && \
+       if [ -n "$ERR" ] ; then echo "go fmt Failed - ERR: ${ERR}" ; exit 1 ; fi && \
```

Comment:
Any particular reason to change these @zededa-yuri ?

Reply:
The explanation is actually in the commit message. Let me give a bit more details. Let's deliberately violate coding style to make `gofmt` complain. I just removed spaces around `*` sign and made definition of `qemuOverHead` look like this:

```go
const qemuOverHead = int64(600*1024*1024)
```

Now if you run `make pkg/pillar` build will produce an error message from which I can not gather what is wrong:

```
 ---> Running in 4831e612d2a8
Running go vet
Running go fmt
go fmt Failed - ERR: ./hypervisor/kvm.go
The command '/bin/sh -c [ -z "$GOARCH" ] || export CC=$(echo /*-cross/bin/*-gcc) ;    echo "Running go vet" && go vet ./... &&     echo "Running go fmt" && ERR=$(gofmt -e -l -s $(find . -name \*.go | grep -v /vendor/)) &&        if [ -n "$ERR" ] ; then echo "go fmt Failed - ERR: "$ERR ; exit 1 ; fi &&     make DISTDIR=/out/opt/zededa/bin build' returned a non-zero code: 1
exit status 1
make: *** [Makefile:648: eve-pillar] Error 1
rm pkg/pillar/Dockerfile
```

It basically says:
> something is wrong with `kvm.go`, keep guessing what I did not like about it

With the proposed patch the failure looks like:
```
 ---> Running in b3ed72d71450
Running go vet
Running go fmt
go fmt Failed - ERR: diff -u ./hypervisor/kvm.go.orig ./hypervisor/kvm.go
--- ./hypervisor/kvm.go.orig
+++ ./hypervisor/kvm.go
@@ -21,7 +21,7 @@

 //TBD: Have a better way to calculate this number.
 //For now it is based on some trial-and-error experiments
-const qemuOverHead = int64(600*1024*1024)
+const qemuOverHead = int64(600 * 1024 * 1024)

 const minUringKernelTag = uint64((5 << 16) | (4 << 8) | (72 << 0))

The command '/bin/sh -c [ -z "$GOARCH" ] || export CC=$(echo /*-cross/bin/*-gcc) ;    echo "Running go vet" && go vet ./... &&     echo "Running go fmt" && ERR=$(gofmt -e -d -s $(find . -name \*.go | grep -v /vendor/)) &&        if [ -n "$ERR" ] ; then echo "go fmt Failed - ERR: ${ERR}" ; exit 1 ; fi &&     make DISTDIR=/out/opt/zededa/bin build' returned a non-zero code: 1
exit status 1
make: *** [Makefile:648: eve-pillar] Error 1
rm pkg/pillar/Dockerfile
```